src/main/resources/db/changelog/alert/5.1.0/changelog-authentication-stored-procedures.xml

```diff
@@ -62,7 +62,7 @@
 
                         statement.execute("CALL ASSIGN_ROLE_PERMISSION_ACCESS('ALERT_JOB_MANAGER', '" + descriptorName + "', 'global', 'READ');");
                         statement.execute("CALL ASSIGN_ROLE_PERMISSION_ACCESS('ALERT_JOB_MANAGER', '" + descriptorName + "', 'global', 'EXECUTE');");
-                        statement.execute("CALL ASSIGN_ROLE_PERMISSION_ALL_ACCESS('ALERT_JOB_MANAGER', '" + descriptorName + "', 'distribution');");
+                        statement.execute("CALL ASSIGN_ROLE_PERMISSION_ALL_NO_UPLOAD_ACCESS('ALERT_JOB_MANAGER', '" + descriptorName + "', 'distribution');");
 
                         statement.execute("CALL ASSIGN_ROLE_PERMISSION_ACCESS('ALERT_USER', '" + descriptorName + "', 'distribution', 'READ');");
```

Comment:
Don't we want User's to have no upload access as well?

Reply:
They only have read access on distribution jobs.